data_retriever/migration_event_queue.py

```
@@ -0,0 +1,204 @@
+from uuid import uuid4
+from enum import Enum
+from psycopg2 import connect as postgres
+from dotenv import load_dotenv
+from os import environ as env, remove as remove_file
+from os.path import exists as path_exists
+from datetime import datetime
+
+from data_retriever.migration_event import deserialize_event, serialize_event, VMShutdownEvent, serialize_event_type, \
```

Ruff (0.12.2)

`data_retriever.migration_event.VMShutdownEvent` imported but unused

Remove unused import: `data_retriever.migration_event.VMShutdownEvent`

(F401)